.github/workflows/reusable_deploy_layer_stack.yml

```diff
@@ -8,15 +8,22 @@ on:
   workflow_call:
     inputs:
       stage:
+        description: "Deployment stage (BETA, PROD)"
         required: true
         type: string
       artifact-name:
+        description: "CDK Layer Artefact name to download"
```

Comment:
nit & minor: I think we are using the American English `artifact` throughout the workflows